app/adapters/contributor.ts

```diff
@@ -1,43 +1,7 @@
-import DS from 'ember-data';
-import Node from 'ember-osf-web/models/node';
 import OsfAdapter from './osf-adapter';
 
-const requestTypes = ['createRecord', 'findRecord'];
-
-export default class Contributor extends OsfAdapter.extend({
-    buildURL(
-        this: Contributor,
-        modelName: 'contributor',
-        id: string,
-        snapshot: DS.Snapshot,
-        requestType: string,
-    ): string {
-        if (requestTypes.includes(requestType)) {
-            return this._super(modelName, id, snapshot, requestType);
-        }
-
-        const nodeId: string = snapshot ? snapshot.record.get('nodeId') : (id.split('-').shift() as string);
-        const node: Node = this.store.peekRecord('node', nodeId);
-
-        if (!node) {
-            throw new Error('Trying to add a contributor to a Node that hasn\'t been loaded into the store');
-        }
-
-        const base: string = this._buildRelationshipURL(
-            node._internalModel.createSnapshot(),
-            'contributors',
-        );
-
-        if (requestType === 'findRecord') {
-            return `${base}${id.split('-').pop()}/`;
-        }
-
-        const sendEmail: boolean | string = snapshot ? snapshot.record.get('sendEmail') : true;
-
-        // Needed for Ember Data to update the inverse record's (the node's) relationship
-        return `${base}?embed=node&send_email=${sendEmail}`;
-    },
-}) {}
+export default class Contributor extends OsfAdapter {
```

Comment:
Cruft removal? Much cleaner looking. Not needed anymore?

Reply:
We don't yet have anything that adds/removes/modifies contributors, so this stuff is unneeded. When we do add contributor management, we should go about it [a different way](https://github.com/CenterForOpenScience/ember-osf-web/pull/197).